spec/javascripts/admin/modules/paste_govspeak.spec.js

```diff
@@ -0,0 +1,55 @@
+describe('GOVUKAdmin.Modules.PasteGovspeak', function () {
+  var textarea, pastedGovspeak
+
+  function createHtmlPasteEvent (html = null) {
+    const event = new window.Event('paste')
+    event.clipboardData = {
+      getData: (type) => {
+        if (type === 'text/html') {
+          return html
+        }
+      }
+    }
+
+    return event
+  }
+
+  beforeEach(function () {
+    textarea = document.createElement('textarea')
+
+    pastedGovspeak = new GOVUKAdmin.Modules.PasteGovspeak()
+    pastedGovspeak.start($(textarea))
+  })
+
+  afterEach(function () {
+    textarea.remove()
```

Comment:
as this isn't attached to the DOM there isn't a need to remove it.

Reply:
done